### DISCUSSION TITLE
Permanent "Interaction Limits"

### DISCUSSION THREAD

**Discussion by macintacos** [community/community#11270] [Other Feature Feedback, Questions, & Ideas]
I just found out about "[Temporary Interaction Limits](https://docs.github.com/en/communities/moderating-comments-and-conversations/limiting-interactions-in-your-repository)". While 6 months is generous, I'd like to have it permanently enabled on some of my repos — any chance that could be added? 🙂

**Comment by ghost** [comment]
I'm going to be honest, it baffles me that this was not considered when the feature was initially added.

**Comment by gekkowrld** [comment]
Or they should at least let people select a custom time frame that you can put let's say 10 years which can be a close to permanent

**Comment by arslivinski** [comment]
I wouldn't mind if they only allow this for paid accounts, at least we would have the option to do this ...
